lib/browser/rpc-server.js

```diff
@@ -457,3 +457,11 @@ ipcMain.on('ELECTRON_BROWSER_SANDBOX_LOAD', function (event) {
     }
   }
 })
+
+ipcMain.on('ELECTRON_BROWSER_GET_LAST_WEB_PREFERENCES', function (event) {
+  try {
+    event.returnValue = event.sender.getLastWebPreferences()
+  } catch (error) {
```

Comment:
Should this error be logged?

Reply:
it was not logged before. I can either log it here or transfer it (use `errorUtils.serialize`) and log it on the renderer process side (use `errorUtils.deserialize`)

Reply:
opted for the 2nd option